include/swift/IDE/SyntaxModel.h

```diff
@@ -97,6 +97,7 @@ enum class SyntaxStructureKind : uint8_t {
   EnumElement,
   TypeAlias,
   Subscript,
+  Call,
```

Comment:
fyi will need a simple change to `ModelASTWalker::walkToDeclPre()` to actually emit this, and I think a possibly less-simple change to `SyntaxModelContext::SyntaxModelContext()` to classify `tok::kw_call` as identifer/keyword depending on context (this affects syntax highlighting).